org.eclipse.dartboard/src/org/eclipse/dartboard/util/Logger.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************

```

Comment:
I don't think we need a seperate Logger class. Can't we just expose the getLogger method and call that in every file? And use org.eclipse.e4.core.services.log.Logger directly?

Reply:
Yes, we can expose the method and call. Calling direct method will show restriction warnings on each file.

Reply:
Ah, I see. Any way around that, except wrapping it in our own Logger? Like is there a plugin logger? If not. We should keep the Logger class.

Reply:
No Plugin is logger available.  
`BundleContext `will be` null ` for the plugins which don't have the `Activator` class. So, not able to set or get logger for our bundle.

Reply:
@vogella could we use an Activator for logging purposes? I really dislike the "incouraged access" or the separate Logger class. Seems like the best way would be an Activator.